packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -253,14 +256,6 @@ export function removeVM(vm: VM) {
     resetComponentStateWhenRemoved(vm);
 }
 
-function getNearestShadowAncestor(vm: VM): VM | null {
```

Comment:
this function walked the owner's chain in search for something that didn't use light dom.